controllers/validators/schema/definitions.js

```diff
@@ -159,6 +159,10 @@ module.exports = {
       required: ['offset', 'limit'],
       additionalProperties: false
     },
+    random: {
+      type: 'string',
+      enum: ['']
```

Comment:
why is enum here?

Reply:
To make sure it's always empty string, i.e. `?filter[random]` and not `?filter[random]=foo`.
Why enum? I tried `const` in the past, but it didn't work for me. Haven't tried this time. We could also use `maxLength=0`.